htcrystalball/examine.py

```diff
@@ -167,16 +167,20 @@ def default_preview(slot_name: str, slot_type: str) -> dict:
 
 
 def rename_slot_keys(slot: dict) -> dict:
-    return {
+    renamed = {
         'node': slot["UtsnameNodename"],
         'type': slot["SlotType"],
         'total_slots': slot["TotalSlots"],
         'cores': slot["TotalSlotCpus"],
-        'gpus': slot["TotalSlotGPUs"],
         'disk': slot["TotalSlotDisk"],
         'ram': slot["TotalSlotMemory"]
     }
 
+    if slot.get('TotalSlotGPUs', None):
```

Comment:
Does htcondor not always return GPU slot information?

Reply:
Actually no, TotalSlotGPUs is only visible in slots that are children to a node with GPUs

Reply:
Ok. Thanks. Next time we have a few extra minutes to argue on Zoom, it would be nice to [bikeshed](https://en.wiktionary.org/wiki/bikeshedding) with everyone about our thoughts on the style here. Should there be a comment; `try` vs `if`.

Reply:
I made this function work for all types of slots (e.g., dynamic, statis, gpu). If I recall correctly I got a key error when calling `TotalSlotGPUs` for a static or dynamic slot, but I'm not 100% sure. I will have to test it.